### PR TITLE
LibTorch 1.10.0 support

### PR DESCRIPTION
**Pull request by NiklasGustafsson**
Adding the new functions, except those in the torch.special module.

### CODE REVIEW

**Comment by NiklasGustafsson** [comment]
Once this is reviewed and merged, we can make a release and push 1.10 support to NuGet.

**Comment by NiklasGustafsson** [comment]
Merging 1.10 support

**Review by dsyme — COMMENTED** [review]
Ooops didn't submit my review

**Comment by NiklasGustafsson** [comment]
@dsyme -- thanks! Since all your comments are about documentation / links, etc., I will follow up with fixes in a future PR.

File: src/TorchSharp/Distributions/Cauchy.cs [hunk 1]
```diff
@@ -31,31 +45,63 @@ public Cauchy(Tensor loc, Tensor scale, torch.Generator generator = null) : base
...
+            /// Returns entropy of distribution, batched over batch_shape.
+            /// </summary>
+            /// <returns></returns>
             public override Tensor entropy()
             {
                 return Math.Log(Math.PI * 4) + scale.log();
             }
+
+            /// <summary>
+            /// Returns the cumulative density/mass function evaluated at `value`.
```
[the remark this answers is not in the document]
Comment (NiklasGustafsson) [reply src/TorchSharp/Distributions/Cauchy.cs:80]: Massive amount of work. Maybe in the future...

File: src/Examples/MNIST.cs [hunk 2]
```diff
@@ -38,7 +38,7 @@ public class MNIST
 
         internal static void Main(string[] args)
         {
-            var dataset = args.Length > 0 ? args[0] : "mnist";
+            var dataset = "mnist"; // args.Length > 0 ? args[0] : "mnist";
```
[the remark this answers is not in the document]
Comment (NiklasGustafsson) [reply src/Examples/MNIST.cs:41]: No.

**Pull request merged**